src/components/Gradebook/index.jsx

```diff
@@ -260,6 +283,16 @@ export default class Gradebook extends React.Component {
                     </label>
                   </span>
                 </div>
+                { this.props.assignmnetTypes.length > 0 &&
+                  <div className="student-filters">
+                    <InputSelect
```

Comment:
Can you provide a label to this drop down? I think you can just call it `Assignment Types`, just like how Studio does it.

Reply:
👍